Index: pygobject-2.28.6/codegen/pygobject-codegen-2.0.in
===================================================================
--- pygobject-2.28.6.orig/codegen/pygobject-codegen-2.0.in	2011-06-13 18:30:25.000000000 +0200
+++ pygobject-2.28.6/codegen/pygobject-codegen-2.0.in	2011-09-17 16:19:42.043397347 +0200
@@ -8,4 +8,4 @@
 PYTHONPATH=$codegendir
 export PYTHONPATH
 
-exec @PYTHON@ $codegendir/codegen.py "$@"
+exec python $codegendir/codegen.py "$@"
